src/ContainerIntegration.jsx

```diff
@@ -10,25 +10,23 @@ import { EmptyStatePanel } from "cockpit-components-empty-state.jsx";
 
 const _ = cockpit.gettext;
 
-export const renderContainerPublishedPorts = (ports) => {
+// ports is a mapping like { "5000/tcp": [{"HostIp": "", "HostPort": "6000"}] }
+export const renderContainerPublishedPorts = ports => {
     if (!ports)
         return null;
 
-    const result = ports.map(port => {
-        // podman v4 has different names than v3
-        const protocol = port.protocol;
-        const hostPort = port.hostPort || port.host_port;
-        const containerPort = port.containerPort || port.container_port;
-        const hostIp = port.hostIP || port.host_ip || '0.0.0.0';
-
-        return (
-            <ListItem key={ protocol + hostPort + containerPort }>
-                { hostIp }:{ hostPort } &rarr; { containerPort }/{ protocol }
-            </ListItem>
-        );
+    const items = [];
+    Object.entries(ports).forEach(([containerPort, hostBindings]) => {
+        (hostBindings ?? []).forEach(binding => {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1350-20230713-051608-1edac15d-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)

Reply:
fixed